doc/spec-files.rst

```diff
@@ -584,7 +584,8 @@ a list of tuples, each tuple having three elements:
 
 * The second element is the script name of the analyzed app (without the ``.py`` extension).
 
-* The third element is the name for the executable (usually the same as the script).
+* The third element is the path to the executable relative to the ``dist``
+  directory (e.g. ``script_dir/script_name``).
```

Comment:
When an executable is a `onedir` executable, the relative path in dist directory is indeed `script_dir/script_name` (usually just `script_name/script_name`). But when the executable is a `onefile` executable, it is just `script_name`. This is an important distinction, and is probably what is throwing off most of the users that run into problems here.